src/main/java/core/basesyntax/Bulldozer.java

```diff
@@ -0,0 +1,13 @@
+package core.basesyntax;
+
+public class Bulldozer extends Machine {
+    @Override
+    public void doWork() {
+        System.out.println("The Bulldozer is doing some work");
```

Comment:
The message 'The Bulldozer is doing some work' could be more informative by specifying what type of work the Bulldozer is performing. Consider providing more details about the task it is doing.